docs/getting-started.md

```diff
@@ -304,8 +311,8 @@ Once you have the local node fully synced, you can run the following command:
 ```
 cargo run -p graph-node --release -- \
   --postgres-url postgresql://<USERNAME><:PASSWORD>@localhost:5432/<POSTGRES_DB_NAME> \
-  --ethereum-rpc <ETHEREUM_NETWORK_NAME>:127.0.0.1:8545`
-  --ipfs 127.0.0.1:5001 \
+  --ethereum-rpc <ETHEREUM_NETWORK_NAME>:127.0.0.1:8545
```

Comment:
There should be a backward slash at the end of this line:
--ethereum-rpc <ETHEREUM_NETWORK_NAME>:127.0.0.1:8545 \